aemedge/scripts/delayed.js

```diff
@@ -15,22 +15,37 @@ function getEnvType(hostname = window.location.hostname) {
   return fqdnToEnvType[hostname] || 'dev';
 }
 
+function getScope(path = window.location.pathname) {
+  return path.startsWith('/news/')
+    || (path.startsWith('/blogs/') && getMetadata('article:tag').includes('content-type/executive-blog'))
```

Comment:
Should this be `getMetadata('article:tag')?.includes('content-type/executive-blog')` as it will fail if there a no tags at all, like for /blogs/ URL.

Reply:
Good catch, will change it. Thanks, @mhaack